src/main/resources/plugin.yml

```diff
@@ -1,21 +1,82 @@
 name: AnarchyExploitFixes
 version: ${project.version}
-main: me.moomoo.anarchyexploitfixes.Main
+main: me.moomoo.anarchyexploitfixes.AnarchyExploitFixes
+prefix: AnarchyExploitFixes
+authors: [ moo, xGinko ]
 description: ${project.description}
 softdepend: [ ProtocolLib ]
-
+api-version: 1.13
+website: https://github.com/moom0o
 commands:
   aef:
-    usage: /<command>
-    description: AnarchyExploitFixes commands
+    usage: /<command> [ reload, version, elytra, geared, lag, config ]
+    description: AnarchyExploitFixes admin commands
   toggleconnectionmsgs:
     usage: /<command>
     description: Toggle connection messages
-
 permissions:
+  anarchyexploitfixes.*:
+    description: Admin plugin permissions
+    children:
+      anarchyexploitfixes.commandwhitelistbypass: true
+      anarchyexploitfixes.chatbypass: true
+      anarchyexploitfixes.netherroofbypass: true
+      anarchyexploitfixes.silentJoin: true
+      anarchyexploitfixes.silentLeave: true
+      anarchyexploitfixes.cmd.*: true
+    default: op
+  anarchyexploitfixes.chatbypass:
+    description: Bypass chat restrictions if enabled.
+    default: op
```

Comment:
Default perm is always op. Waste of space.

Reply:
Thanks